spec.html

```diff
@@ -1448,11 +1449,12 @@ <h1>Numeric Types</h1>
               _T_::sameValueZero(x,&nbsp;y)
             </td>
             <td>
+              `[x].includes(y)`
```

Comment:
this seems much less clear than `x === y` - which is recursive, but so is `.includes`
```suggestion
              `x === y`
```

Reply:
`x === y` does not use `sameValueZero` - `===` considers `NaN` as being unequal to itself. `[NaN].includes(NaN)` holds, though, and so is appropriate here.

Reply:
ah, right, forgot about NaN.

This doesn't link to `Array.prototype.includes` though, just like `T::sameValue` etc don't link - so i'm not sure how much clearer this is if i still have to find-in-page to understand the difference?

Reply:
A reader might remember how the JS operations work but not the spec AOs.

Besides, these were the only two empty cells in this column - might as well fill them out.

Reply:
fair enough